op-program/client/l2/test/miner.go

```diff
@@ -67,9 +67,6 @@ func NewMiner(t *testing.T, logger log.Logger, isthmusTime uint64) (*Miner, *cor
 	}
 	n, err := node.New(nodeCfg)
 	require.NoError(t, err)
-	t.Cleanup(func() {
-		_ = n.Close()
-	})
```

Comment:
Doesn't this leave the node and backend running? In running locally I haven't ever seen it fail if we remove the `t.Parallel()` from the tests so they run sequentially. It does make it slower but better than crashing.   I still don't understand why it crashes though.  This does seem to help but probably isn't the right answer to leave this unclosed.

Reply:
So interestingly we never actually started the node so possibly just not closing it is reasonable, but it's a little hard to tell.

The other option I think would be https://github.com/ethereum-optimism/optimism/pull/15289 which fills out the geth config structs some more. Based it on the action test setup since we know that runs reliably. I'm actually not really sure if we have to start the node before using it or if skipping Start and Close is a lighter weight but still safe option.  It does seem weird to use something without starting it though...